eZ/Publish/API/Repository/Tests/Regression/EZP28046RedirectLoopWhenAccessingChildrenAfterRenameParentTest.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\API\Repository\Tests\Regression;
+
+use eZ\Publish\API\Repository\Tests\BaseTest;
+
+/**
+ * Issue https://jira.ez.no/browse/EZP-28046.
+ * @group regression
+ * @group ezp28046
+ * @group cache
+ * @group cache-invalidation
+ * @group cache-spi
+ */
+class EZP28046RedirectLoopWhenAccessingChildrenAfterRenameParentTest extends BaseTest
+{
+    public function testEZP28046()
```

Comment:
IMHO this should be a part of `UrlAliasServiceIntegrationTest` and should be `testLookupOnRenamedParent`. 
Yes, I've seen regression tests before in kernel package, but I'm rather against such naming convention, especially when this looks like completely sane test case.

However if @andrerom disagrees, then I'll approve :)

Reply:
Either way is fine with me, but it's indeed a bit cleaner what you propose here @alongosz